CHANGES_NEXT_RELEASE

```diff
@@ -28,3 +28,4 @@
 - Fix: Empty string detection for entity id, entity type and attribute name in NGSIv2 request URLs (#1426, #1487)
 - Fix: Trailing slashes in URI path no longer silently removed, but considered and giving adequate error responses
 - Fix: Avoid rendering at the same time id and idPattern fields in NGSIv2 subscriptions
+- Fix: Error returned on GET/DELETE requests specifying Content-Type (Issue #2128)
```

Comment:
It seems that for #1887, #1890 and #1909 the entry is already in CNR but the issue number has not been included (as the fix was done without realizing that an issue about that exists... funny :)

Thus, I think we could take this opportunity to edit previous CNR lines in the following way:

* Fix: Empty string detection for entity id, entity type and attribute name in NGSIv2 request URLs (#1426, #1487) -> add #1887
* Fix: NGSIv2 "entity does not have such attribute" error case was wrongly using "requested entity has not been found" descriptions -> add #1890
* Fix: missing error payload for NotFound error in 'PUT /v2/entities/<eId>/attrs/<attrName>' and 'PUT /v2/entities/<eId>/attrs/<attrName>/value' operations -> add #1909 

What do you think?